python/tvm/relay/ir_pass.py

```diff
@@ -34,3 +34,5 @@ def infer_type(env, expr):
 free_vars = _ir_pass.free_vars
 
 free_type_vars = _ir_pass.free_type_vars
+
+dead_code_elim = _ir_pass.dead_code_elim
```

Comment:
do not do this anymore, instead do

```python

def dead_code_elimination(expr):
    """
    document here
    """
    return _ir_pass.dead_code_elimination(expr)
```

Reply:
I would fix the rest in a seprate pr, is it ok with you?